core/src/main/java/org/modelmapper/internal/ProxyFactory.java

```diff
@@ -87,6 +89,7 @@
 
     try {
       final DynamicType.Unloaded<T> unloaded = new ByteBuddy()
+		      .with(new NamingStrategy.SuffixingRandom("ByteBuddy", NO_PREFIX))
```

Comment:
It would be better if we can align the indention!

Reply:
Completely agree. Going to change it right now.